src/wrapper/strings/ffi_str.rs

```diff
@@ -78,8 +78,11 @@ impl JNIString {
 }
 
 impl JNIStr {
-    /// Construct a reference to a `JNIStr` from a pointer. Equivalent to
-    /// `CStr::from_ptr`.
+    /// Construct a reference to a `JNIStr` from a pointer. Equivalent to `CStr::from_ptr`.
+    ///
+    /// # Safety
+    ///
+    /// Expects a valid pointer to a valid immutable C string.
```

Comment:
Strings in C can't be mutable, can they? So I suggest
```suggestion
    /// Expects a valid pointer to a null-terminated C string and does not perform any lifetime checks for the resulting value.
```